src/ColumnsBlock/ColumnsBlockView.jsx

```diff
@@ -20,6 +21,7 @@ const getSides = (v) => {
 };
 
 const ColumnsBlockView = (props) => {
+  const location = useLocation() || {};
```

Comment:
@tiberiuichim Tests are failing with:

```
  ● renders 2 columns



    TypeError: Cannot read properties of undefined (reading 'location')



      22 |
      23 | const ColumnsBlockView = (props) => {
    > 24 |   const location = useLocation() || {};
```

Reply:
@avoinea @tiberiuichim I tried to run the tests locally with the `yarn test` command, but I get the error:

```bash
➤ YN0000: └ Completed in 6s 686ms
➤ YN0000: ┌ Post-resolution validation
➤ YN0028: │ The lockfile would have been modified by this install, which is explicitly forbidden.
➤ YN0000: └ Completed
➤ YN0000: Failed with errors in 6s 837ms
make: *** [Makefile:46: test] Erro 1
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Any tips on how to solve this problem?

Reply:
I think the testing infrastructure should be updated. I know @sneridagh had a fix for this kind of problem.

Reply:
I researched this problem and, from what I understood, it occurs when the node configuration:

`enableImmutableInstalls`

is `true`

and there is a change in the `yarn.lock` file when we run yarn: See:

https://stackoverflow.com/questions/67062308/getting-yn0028-the-lockfile-would-have-been-modified-by-this-install-which-is-e

But the probability of `yarn.lock` being changed is very high. For example, the addon may have a specific dependency. So I ask if we shouldn't force `enableImmutableInstalls=false` in `volto-addon-ci`.

Reply:
I opened an issue on volto-addon-ci:

https://github.com/plone/volto-addon-ci/issues/11

Reply:
I'd go with that fix, yes. I explained it in there.

Reply:
@avoinea @tiberiuichim I fix tests in #59

Reply:
@tiberiuichim I think this PR can be closed since that the #59 has been merged.